<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transfer</class>
 <widget class="QWidget" name="Transfer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>529</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>529</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>529</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>529</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="background.qrc">:/image/Better Template Forms.jpg</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>80</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="firsNameLastNameLayout">
    <item>
     <widget class="QLabel" name="firstNameLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 14pt &quot;Myanmar Text&quot;;</string>
      </property>
      <property name="text">
       <string>First Name </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lastNameLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 14pt &quot;Myanmar Text&quot;;</string>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="nationalCodeLabel">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>122</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 11pt &quot;Myanmar Text&quot;;</string>
   </property>
   <property name="text">
    <string>National Code</string>
   </property>
  </widget>
  <widget class="QLabel" name="transferLabel">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>20</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 700 30pt &quot;Myanmar Text&quot;;</string>
   </property>
   <property name="text">
    <string>Transfer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backPushButton">
   <property name="geometry">
    <rect>
     <x>608</x>
     <y>125</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(4, 27, 48);
color: rgb(255, 255, 255);
font: 12pt &quot;Myanmar Text&quot;;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="yourBalanceLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>215</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 700 16pt &quot;Myanmar Text&quot;;</string>
   </property>
   <property name="text">
    <string>Your Balance : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>301</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font:  16pt &quot;Myanmar Text&quot;;</string>
   </property>
   <property name="text">
    <string>value</string>
   </property>
  </widget>
  <widget class="QLabel" name="distinationCardNumberLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 700 16pt &quot;Myanmar Text&quot;;</string>
   </property>
   <property name="text">
    <string>Distination Card Number :  </string>
   </property>
  </widget>
  <widget class="QComboBox" name="originCardNumberLabel">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>294</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(4, 27, 48);
color: rgb(255, 255, 255);
font: 14pt &quot;Myanmar Text&quot;;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backPushButton_2">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>450</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(4, 27, 48);
color: rgb(255, 255, 255);
font: 12pt &quot;Myanmar Text&quot;;</string>
   </property>
   <property name="text">
    <string>Transfer</string>
   </property>
  </widget>
  <widget class="QLabel" name="distinationCardNumberLabel_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>291</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 700 16pt &quot;Myanmar Text&quot;;</string>
   </property>
   <property name="text">
    <string>Origin Card Number :  </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="distinationCardNumberLineEdit">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>340</y>
     <width>255</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Myanmar Text&quot;;
background-color: rgb(4, 27, 48);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
